### ISSUE TITLE
How to include glue artifacts in notebooks that aren’t built in the book

### ISSUE THREAD

**Issue by choldgraf** [ExecutableBookProject/MyST-NB#80]
> Another use case that might be worth documenting, is that now you could have a notebook(s) that doesn't actually show up in the documentation (you signify this to sphinx by adding "orphan: True" in the notebook metadata, see file-wide-metadata), then you can have a separate markdown file that uses these roles/directives to show selected figures/data from one or more notebooks.

> I've talked about this use case before; when you want to use the notebook(s) as a "log" of your data analysis, then have a separate document which is the final "publishable" output.

*originally from @chrisjsewell in #77 *

**Comment by choldgraf** [comment]
I think another option would be to allow for a config variable like “glue_notebooks” that could point to a folder of notebooks and load then all into the glue cache.

**Comment by chrisjsewell** [comment]
> I think another option would be to allow for a config variable like “glue_notebooks” that could point to a folder of notebooks and load then all into the glue cache.

Yes but the advantage of having these notebooks within the sphinx source folder, is that they will be integrated in the jupyter-cache execution (#55)

**Comment by choldgraf** [comment]
Makes sense - I'm also trying to think in terms of the CLI as well. E.g. with a global TOC how would this be accomplished? (I suppose that notebook-level metadata like `:orphan:` could still work in this case)

**Comment by chrisjsewell** [comment]
I’ll leave that for you to figure out 😉
